brightcove/brightcove-popup/src/components/modal.tsx

```diff
@@ -128,19 +128,30 @@ const Modal: React.FC<ModelProps> = (props) => {
     setIsSelected((prevSelected) => !prevSelected);
   };
 
+  const setQueryHandler = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const query = event.currentTarget.value;
+    setSearchQuery(query);
+  };
+
   const searchQueryHandler = async (
     event: React.KeyboardEvent<HTMLInputElement>
   ) => {
-    const query = event.currentTarget.value.toLowerCase();
-    setSearchQuery(query);
+    const query = event.currentTarget.value;
 
     if (event.code === 'Enter' && brightcove) {
       try {
+        const {
+          data: { count },
+        } = await brightCove.getVideos({
+          authUrl: config.oauthUrl,
+          videoUrl: `${config.videocountUrl}?q=${query}&limit=${limit}&offset=0`,
+        });
         const queryVideos = await brightcove.getVideos({
           authUrl: config.oauthUrl,
           videoUrl: `${config.searchUrl + query}&limit=${limit}&offset=0`,
         });
         setOffset(0);
+        setCount(count);
         setRenderVideos(queryVideos.data);
         setErrorFound(queryVideos.data.length === 0 ? true : false);
```

Comment:
No need to add conditional operator here
`queryVideos.length` give `true` or `false`